src/org/labkey/targetedms/TargetedMSController.java

```diff
@@ -3454,18 +3450,18 @@ public ModelAndView getView(SkylineAuditLogExtraInfoForm form, BindException err
         {
             SkylineAuditLogExtraInfoBean bean;
             String errMessage = null;
-            AuditLogEntry ent = AuditLogEntry.retrieve(form.getEntryId());
+            AuditLogEntry ent = AuditLogEntry.retrieve(form.getEntryId(), getViewContext());
```

Comment:
I need both user and container to run a query so I just pass the view context instead of container only.